drivers/video/vnc/vnc_fbdev.c

```diff
@@ -619,6 +622,22 @@ int up_fbinitialize(int display)
       gerr("ERROR: wait for vnc server start failed: %d\n", ret);
     }
 
+  /* Save the input callout function information in the session structure. */
+
+  session           = g_vnc_sessions[display];
+#ifdef CONFIG_VNCSERVER_TOUCH
+  session->touch    = vnc_touch_register(CONFIG_VNCSERVER_TOUCH_DEVNAME);
```

Comment:
should we call vnc_touch_register before vnc_wait_start? otherwise, the background thread may reference touch before we register the driver.

Reply:
OK, moved before vnc_wait_start